test/scholar/linear/logistic_regression_test.exs

```diff
@@ -0,0 +1,168 @@
+defmodule Scholar.Linear.LinearRegressionTest do
+  use ExUnit.Case, async: true
+
+  describe "Logistic Regression" do
```

Comment:
We don't need this block, since everything here is about logistic regression :D

Reply:
The module name is incorrect, by the way
LinearRegression -> LogisticRegression